### PR TITLE
commiting warmup task 01 to 04

### PR DESCRIPTION
**Pull request by vyku**

### CODE REVIEW

**Comment by vyku** [comment]
pull request for warmup task week 3

**Comment by cheuschober** [comment]
retest this please

File: task_07.py [hunk 1]
```diff
@@ -5,3 +5,11 @@
 import task_06
 
 WORDS = task_06.WORDS
+
+if "GRANARIES" in WORDS:
```
Comment (cheuschober) [inline task_07.py]: GRANARIES_EXIST = 'granaries' in WORDS

File: task_01.py [hunk 2]
```diff
@@ -7,3 +7,9 @@
 
 
 RAVEN = 'quoth'
+
+RAVEN = 'nevermore!'
+
+print RAVEN
```
Comment (cheuschober) [inline task_01.py]: String not capitalized correctly.

File: task_02.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!usr/bin/env python
+# _*_ coding: utf _8 _*_
+
+""" a sample doc string """
+
+
+week =(((19 % 10) + 100 )+(2 ** 8))/7
+
+print week
+
```
Comment (cheuschober) [inline task_02.py]: extra whitespace, variable named incorrectly.

File: task_03.py [hunk 4]
```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-"""Slice this string."""
+
+WILL_ROBINSON = "Danger Will Robinson!"
+
+
+KLAXON = WILL_ROBINSON[0:7]
```
Comment (cheuschober) [inline task_03.py]: Good use of slice but extra whitespace caused lint violations.

**Pull request merged**